examples/terminating-gateway-tls/efs.tf

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+resource "aws_security_group" "efs" {
+  name        = "${var.name}-efs-sg"
+  description = "Allows inbound efs traffic from ec2"
+  vpc_id      = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 2049
+    to_port     = 2049
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
This should include the server app's SG as the source security group. Let's not expose stuff to the internet